README.md

```diff
@@ -44,12 +44,24 @@ To get started:
    pre-commit install
    ```
 
-5. Run the web app:
+5. Run the webapp:
 
    ```bash
    python manage.py runserver
    ```
 
+   When running the webapp for the first time you may get a warning similar to:
+
+   `You have 19 unapplied migration(s). Your project may not work properly until you apply the migrations for app(s): admin, auth, contenttypes, main, sessions.`
+
+   If this is the case, stop your webapp (with CONTROL/CMD-C) and apply the migrations with:
```

Comment:
This is one of the few times you actually do use the CONTROL key on a Mac instead of CMD

```suggestion
   If this is the case, stop your webapp (with CONTROL-C) and apply the migrations with:
```